packages/liveblocks-node/src/client.ts

```diff
@@ -70,10 +115,10 @@ export class Liveblocks {
 
   /** @internal */
   private async post(
-    path: `/${string}`,
+    relativeURL: string,
     json: Record<string, unknown>
   ): Promise<Response> {
-    const url = urljoin(this._baseUrl, path);
+    const url = new URL(relativeURL, this._baseUrl);
```

Comment:
@nvie You'll notice `urljoin` has been replaced with `new URL()` here. I was getting some weird errors when using `urljoin` and making a fetch request because it was encoding characters like `?` but the API handler wasn't  reading it correctly.

Here is a reproduction of the issue:  [Playground link](https://www.typescriptlang.org/play?ssl=8&ssc=1&pln=8&pc=71#code/GYVwdgxgLglg9mABCATgGwFZxmAFAIwEMBnAUwFV0AuRYqFHAc0QB9FyAlAGQBpEAHQlAAWNOgzCMAlGPpNEAbwBQiRBAR1k6RAF5EYUgHd23AiQropAbhVa0AOkEiwhALaldAocJuqUpKFQkVAcoOABlOUlcayUAXyUldTBiODRSezQ4RlwQrBxcAHJhKCh+YioAekrCfhhMmAA3UnwsiABrYnt4Qr5CysaAJkqUODhXYgB+NBhXGCgdQcKpKSSNNIysnINjTi4igeHR8amZuYWlvpKyiura+pnm1rgOrp6VoA)

I also think using `URL` directly simplifies things here. Let me know what you think?